pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LinuxInfoUtils.java

```diff
@@ -60,6 +62,7 @@ public class LinuxInfoUtils {
     private static Method getCpuQuotaMethod;
     private static Method getCpuPeriodMethod;
     private static Method getCpuUsageMethod;
+    private static Set<String> exposeNicLimitFaultCompleted = new HashSet<>();
```

Comment:
Logs are always archived, so printing them only once is not conducive to troubleshooting.
Maybe we can change it to ` if(++counter % 1000 == 0)  log.error(...)`

Reply:
Change as suggested